### DISCUSSION TITLE
License disclaimer field

### DISCUSSION THREAD

**Discussion by elibroftw** [ironSource/license-report#140] [Q&A]
Just wondering how to get the actual license each project uses? I want to make an accordion for each project with the content being its actual license rather than just a link to the project.

**Comment by BePo65** [comment]
What do you mean with the term "actual license"?

If you want the license the package runs under (e.g."MIT") then take a look at the output field "licenseType".

If you want the text of the license the package runs under, then take a look at the project [License-downloader](https://github.com/BePo65/license-downloader), which will get you the text of all licenses from the json result of running license-report.
